### PR TITLE
[Wait for #1197] [Test/Model] Prepare addition model

### LINKED ISSUE #1213: Realize Multiout is not setting correctly
multiout layer is parsed before output_layer is set, so realizeMultiout is doing nothing

### PR DESCRIPTION
**Pull request by zhoonit**
- [Test/Model] Prepare addition model

```
This patch add addition model test

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

~- **[Pending MERGE commit: #1188]** Merge branch '"'"'graph_core_moved'"'"' into as/validate/non_linears/2_base~

fixes #1213

### CODE REVIEW

File: nntrainer/graph/network_graph.cpp [hunk 1]
```diff
@@ -488,19 +489,35 @@ int NetworkGraph::realizeGraph() {
...
     // Flatten in TimeDistLayer is not supported.
     if (l.getFlatten() && l.getType() != TimeDistLayer::type) {
       status = realizeFlattenType(l);
       NN_RETURN_STATUS();
     }
   }
 
-  setOutputLayers();
+  try {
+    setOutputLayers();
```
Comment (kparichay) [inline nntrainer/graph/network_graph.cpp:500]: can this be simply moved to line 455, and then call here once again?

Comment (zhoonit) [reply nntrainer/graph/network_graph.cpp:500]: You mean before realizing activation and others? In that case, we have to take care about output layers as well.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**